spec/hanami/view_spec.rb

```diff
@@ -381,19 +381,17 @@ class ConfigurationLayout
       end
 
       it 'a view inherits the configuration from the framework' do
-        pending('Does == mean something different in rspec and minitest?')
         expected = Hanami::View.configuration
         actual   = ConfigurationView.configuration
 
-        expect(actual).to eq(expected)
+        expect(actual.root).to eq(expected.root)
```

Comment:
Sadly, that does not work. I think `expect(actual == expected).to be(true)` is equivalent to `expect(actual).to eq(expected)`.

Also makes sense while debugging:

```
[384, 393] in /Users/ferdi/vendor/hanami/view/spec/hanami/view_spec.rb
   384:         expected = Hanami::View.configuration
   385:         actual   = ConfigurationView.configuration
   386:
   387:         require 'byebug'; byebug
   388:
=> 389:         expect(actual == expected).to be(true)
   390:       end
   391:
   392:       it 'a view inherits the parent' do
   393:         parent = ConfigurationView.configuration
(byebug) expected
#<Hanami::View::Configuration:0x007fc1e5c86690 @namespace=Object, @root=#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>, @default_encoding=#<Encoding:UTF-8>, @partials={}, @views=#<Set: {ConfigurationView}>, @layouts=#<Set: {ConfigurationLayout}>, @load_paths=#<Hanami::Utils::LoadPaths:0x007fc1e5c859c0 @paths=[#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>]>, @layout=nil, @modules=[]>
(byebug) actual
#<Hanami::View::Configuration:0x007fc1e5c85038 @namespace=Object, @root=#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>, @default_encoding=#<Encoding:UTF-8>, @partials={}, @views=#<Set: {}>, @layouts=#<Set: {}>, @load_paths=#<Hanami::Utils::LoadPaths:0x007fc1e5c84980 @paths=[#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>]>, @layout=nil, @modules=[]>
```

There are differences in the attributes of the configurations. However, when running the same test from before migration to rspec:

```
[37, 46] in /Users/ferdi/vendor/hanami/view/test/view_test.rb
   37:       expected = Hanami::View.configuration
   38:       actual   = ConfigurationView.configuration
   39:
   40:       require 'byebug'; byebug
   41:
=> 42:       actual.must_equal(expected)
   43:     end
   44:
   45:     it 'a view inheriths the parent' do
   46:       parent = ConfigurationView.configuration
(byebug) expected
#<Hanami::View::Configuration:0x007ff84925d8e0 @namespace=Object, @root=#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>, @default_encoding=#<Encoding:UTF-8>, @partials={}, @views=#<Set: {ConfigurationView}>, @layouts=#<Set: {ConfigurationLayout}>, @load_paths=#<Hanami::Utils::LoadPaths:0x007ff84925c760 @paths=[#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>]>, @layout=nil, @modules=[]>
(byebug) actual
#<Hanami::View::Configuration:0x007ff849257d00 @namespace=Object, @root=#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>, @default_encoding=#<Encoding:UTF-8>, @partials={}, @views=#<Set: {}>, @layouts=#<Set: {}>, @load_paths=#<Hanami::Utils::LoadPaths:0x007ff8492573f0 @paths=[#<Pathname:/Users/ferdi/vendor/hanami/view/spec/support/fixtures/templates>]>, @layout=nil, @modules=[]>
```

There seem to be the same kind of differences. So it looks like the premise of the test is wrong? I'm a bit confused by this...